dlp/snippets/deid/deid_test.go

```diff
@@ -570,3 +570,28 @@ func TestReidTextDataWithFPE(t *testing.T) {
 		t.Errorf("reidentifyFreeTextWithFPEUsingSurrogate got %q, want %q", got, want)
 	}
 }
+
+func TestDeIdentifyTableWithCryptoHash(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	var buf bytes.Buffer
+	transientKey := "YOUR_TRANSIENT_CRYPTO_KEY"
```

Comment:
Please use transientKeyName both variable name(s) and text.

Reply:
Updated.